cmd/akash/provider.go

```diff
@@ -152,7 +153,11 @@ func doProviderRunCommand(session session.Session, cmd *cobra.Command, args []st
 
 	if ok, _ := cmd.Flags().GetBool("kube"); ok {
 		session.Log().Debug("using kube client")
-		cclient, err = kube.NewClient(session.Log().With("cmp", "cluster-client"))
+		ns, err := cmd.Flags().GetString("manifest-ns")
+		if err != nil {
+			return err
+		}
+		cclient, err = kube.NewClient(session.Log().With("cmp", "cluster-client"), ns)
```

Comment:
add the argument `"{{ template "akash-provider.name" . }}"` [here](https://github.com/ovrclk/akash/blob/46c1e48b712fc9b21912940be4c3a2a57489073d/_run/multi/akash-provider/templates/deployment.yaml#L27) and add `--manifest-ns "$1"` [here](https://github.com/ovrclk/akash/blob/46c1e48b712fc9b21912940be4c3a2a57489073d/_run/multi/akash-provider/run.sh#L39)